### DISCUSSION TITLE
Setting a Cutting Set to an element

### DISCUSSION THREAD

**Discussion by LPeccoud** [cwapi3d/cwapi3dpython#157] [Q&A]
Hello,
How can i set a cuttting_set to an element ?
Reverse function of attribute_controller get_cutting_set(element_id)
Thanks

**Comment by Brunner246** [comment]
There is atm `cut_elements_with_overmeasure` which uses the active set. 
We'll add a method to set a cutting set.

**Comment by Brunner246** [comment]
https://github.com/cwapi3d/cwapi3dpython/issues/167
